dev-client/src/screens/LocationScreens/components/SoilIdSections.tsx

```diff
@@ -76,6 +78,7 @@ export const SoilIdMatchesSection = ({siteId, coords}: SoilIdSectionProps) => {
           <SiteScoreInfoContent
             locationMatch={locationMatch}
             dataMatch={dataMatch}
+            soilData={soilData}
```

Comment:
question: since each of these constants are only used once, does it make sense to pass the imported values directly?

Reply:
This is placeholder code; the actual data will be passed in this way since it needs to be loaded fromt he server.